utils/build/docker/php/apache-mod-7.0.Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG PHP_VERSION=7.0
+ARG VARIANT=release-zts
+
+FROM datadog/dd-appsec-php-ci:php-$PHP_VERSION-$VARIANT
+
+ENV DD_TRACE_ENABLED=1
+ENV DD_TRACE_GENERATE_ROOT_SPAN=1
+ENV DD_TRACE_AGENT_FLUSH_AFTER_N_REQUESTS=0
+ENV DD_TRACE_HEADER_TAGS=user-agent
```

Comment:
**HELP WANTED**

I am trying to remove as much duplication as possible from all the dockerfiles. These 4 lines are duplicated in all dockerfiles and I would like to move them to `utils/build/docker/php/apache-mod/build.sh`. I have tried to add them to the `.zshrc` file and also to add a SH file into the `profile.d` folder but none of them worked.

Reply:
In fact, all dockerfiles are 99% the same. The only thing which changes is the first line where the PHP version is specified. Ideally, calling a generic dockerfile with a build argument which specifies the version would be good to avoid duplication. That said, current duplication is not the end of the world. Any idea is welcome

Reply:
Hi,

You can't set a env var by calling an `.sh` in a dockerfile.

Though, common env var can be set in `utils/build/docker/set-system-tests-weblog-env.Dockerfile`, **if and only if** they are implemented the sane in all tracers. It's not the case for instance for DD_TRACE_HEADER_TAGS.

That being said, we can imagine modifying `utils/build/build.sh ` to have a common dockerfile for each tracer : 

on line 119 , somthing like 

```
IF utils/build/docker/${TEST_LIBRARY}/set-common.Dockerfile EXISTS THEN
        docker build \
            --progress=plain \
            -f utils/build/docker/${TEST_LIBRARY}/set-common.Dockerfile \
            -t system_tests/weblog \
            $EXTRA_DOCKER_ARGS \
            .
fi
```

And you set in utils/build/docker/${TEST_LIBRARY}/set-common.Dockerfile eveything common across all your weblog

Reply:
Thanks for info about the env vars!

Regarding the common dockerfile, how would it get the version of php that it needs to build?

Reply:
it doesn't need to have it. Or did I missed something ?

Reply:
Just to keep this PR up to date. We had a call and spoke about this. The agreed is that there will be have a kinda dockerfile extending weblog for common things within a language. That will avoid duplication. This will be done in a different PR when FPM gets done(so we can provide a robust solution which works with both)